opensquirrel/writer/writer.py

```diff
@@ -4,7 +4,7 @@
 
 
 class _WriterImpl(IRVisitor):
-    number_of_significant_digits = 8
+    FLOAT_PRECISION = 8
```

Comment:
Shouldn't his be `FIXED_POINT_PRECISION`, since we are using this to convert the floating point to a fixed point?

Reply:
I don't think so.

1. We are not converting anything. We are just formatting an output with a given precision.
2. Fixed point and floating point are two different formats for representing a real number. But here we are dealing with `IR.Float`, which contains a Python's `float`, which is a floating point number. So any reference to fixed point is misleading here, IMO.

Python's `round` function names this precision argument as `ndigits`. numpy uses the term `decimals`. Quantify Scheduler exporter uses the term `DEG_PRECISION` (radian to degree conversion outcome precision)....

What do you think of calling it `FLOAT_OUTPUT_DECIMALS`, or `FLOAT_OUTPUT_NDIGITS`?